src/apis/treasury.ts

```diff
@@ -1,13 +1,26 @@
 import { AccountId, Balance } from "@polkadot/types/interfaces/runtime";
 import { ApiPromise } from "@polkadot/api";
+import { sendLoggedTx } from "../utils";
+import { AddressOrPair } from "@polkadot/api/submittable/types";
+import { EventRecord } from "@polkadot/types/interfaces";
 
 export interface TreasuryAPI {
     totalPolkaBTC(): Promise<Balance>;
     balancePolkaBTC(id: AccountId): Promise<Balance>;
+    transferPolkaBTC(destination: string, amount: String): Promise<void>;
```

Comment:
I re-checked the type of the `api.tx.polkaBtc.transfer` and the second parameter (the `amount`) is of type `Compact<Balance> | AnyNumber | Uint8Array`, where `AnyNumber = BN | BigInt | Uint8Array | number | string;`. So the call should work with the lowercase `string` type